confidence/__init__.py

```diff
@@ -1,10 +1,10 @@
 from confidence.exceptions import ConfigurationError, ConfiguredReferenceError, MergeConflictError, NotConfiguredError
-from confidence.io import DEFAULT_LOAD_ORDER, load, load_name, loaders, loadf, loads, Locality
+from confidence.io import DEFAULT_LOAD_ORDER, dumpf, dumps, load, load_name, loaders, loadf, loads, Locality
```

Comment:
I wouldn't really miss it, but why isn't there a `dump` that takes a file handle?

Reply:
Hmm, should be easy to add, not sure it'd see any use in practice :thinking: 
I don't see it being useful with an actual file, but maybe someone comes with a quirky use case where something file-like is preferred. I'll add it :)

Reply:
Ah thanks. I share the opinion that I don't see the use case for myself directly. I was mostly interested if there was a reason for it not being there.

Reply:
`yaml.safe_dump` will support it, so it's easy enough :)